hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/OzoneClientConfig.java

```diff
@@ -144,6 +144,23 @@ public enum ChecksumCombineMode {
       tags = ConfigTag.CLIENT)
   private int retryInterval = 0;
 
+  @Config(key = "max.read.retries",
+      defaultValue = "3",
+      description = "Maximum number of retries by Ozone Client on "
+          + "encountering connectivity exception when reading a key.",
+      tags = ConfigTag.CLIENT)
+  private int maxReadRetryCount = 3;
+
+  @Config(key = "read.retry.interval",
```

Comment:
Currently retry interval is not effective in read retry case.
This has been fixed in feature branch HDDS-7593 and needs to be merged in master.
https://github.com/apache/ozone/pull/6292

Reply:
@ashishkumar50 Looks like it has already been merged.

Reply:
Yes it's merged but currently in feature branch now. Later it will be merged in master. Anyway this patch is to make configurable and that is fine.